apps/openmw/mwmechanics/spells.hpp

```diff
@@ -118,6 +114,10 @@ namespace MWMechanics
 
             void readState (const ESM::SpellState& state, CreatureStats* creatureStats);
             void writeState (ESM::SpellState& state) const;
+
+            bool setSpells(const std::string& id);
+
+            void init(const std::vector<std::string>& spells);
```

Comment:
init is a really odd name for a (public) function in C++, since the actual init phase of construction is supposed to be done in the constructor. To be clear, I do not have a problem with the function per se, just with the name. Maybe you could find something make explicit?

Reply:
Changed it to `addAllToInstance` to distinguish it from a hypothetical `addAll` that is equivalent to calling `add` in a loop. Doesn't really roll off the tongue, but hey.